app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -41,7 +41,10 @@ def cached_api
       end
 
       def configuration_params
-        params.require(:super_good_solidus_taxjar_configuration).permit(:preferred_reporting_enabled)
+        params_hash = params.require(:super_good_solidus_taxjar_configuration).permit(:preferred_reporting_enabled, :preferred_reporting_enabled_at_integer)
+        {
+          preferred_reporting_enabled_at_integer: params_hash[:preferred_reporting_enabled] ? DateTime.now.to_i : nil
```

Comment:
I am a bit confused about this. Is this the date we will be using for back-filling transactions? If so I don't think we want to rely on someone hitting the enable button at the time when they want that to start?

If I recall the backfill dashboard has a date picker, should we be basing it on that instead or using this date there 😕